### DISCUSSION TITLE
Diagrama de clases

### DISCUSSION THREAD

**Discussion by federico42o** [FrancoGL/ISPC-FullStack#94] [Show and tell]
### Buenas estuve adaptando un poco el diagrama del otro dia y llegue a esto, creo que ya se acerca bastante al diagrama final, diganme que opinan asi lo agregamos al IEEE830
*** 
![destinoCordoba drawio](https://user-images.githubusercontent.com/50332167/233159132-5f343653-28f1-4315-b8f4-c92e19966a70.png)

aca dejo el link por si quieren agregar/quitar algo 
https://drive.google.com/file/d/1Ny99Ctof692s7kzx0TacgkbyTLfwkiMY/view?usp=sharing

**Comment by ZafkielEdt** [comment]
Sí, este está más completo. Además de colocarlo en el IEEE-830, habría que cambiarlo por el que deje en la wiki xd

**Reply by federico42o** [reply]
dale ahora lo actualizo
